src/backend/columnar/cstore_customscan.c

```diff
@@ -162,6 +162,12 @@ CStoreSetRelPathlistHook(PlannerInfo *root, RelOptInfo *rel, Index rti,
 	Relation relation = RelationIdGetRelation(rte->relid);
 	if (relation->rd_tableam == GetColumnarTableAmRoutine())
 	{
+		if (rte->tablesample != NULL)
+		{
+			ereport(ERROR, (errcode(ERRCODE_FEATURE_NOT_SUPPORTED), errmsg(
+								"sample scans not supported on columnar tables")));
```

Comment:
nit; for readability of error messages I would suggest to have the `errcode` and `errmsg` line up like below.

```suggestion
			ereport(ERROR, (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
							errmsg("sample scans not supported on columnar tables")));
```

^ ps: tab spaces in our code and github do not align so this might not pass our formatter

Reply:
I ran it through citus_indent, does that not handle these cases?

Reply:
Done.

Reply:
thanks for the change.

for background:

`citus_indent` breaks at unexpected places some times. Like here it broke the line at the start of the string instead of moving the whole next statement to the next line. Which is probably adhering to the rules configured -doh, its a machine-, but makes it harder for humans to read.

Same could happen with the type and the name of a variable. Sometimes `citus_indent` would break them apart if they are in the argument list of a function with a long name, which makes it very unpleasant to the eyes.

I often scan the changes `citus_indent` makes by staging all my changes before running `citus_indent` to see where it breaks.